DEPLOYMENT_FULL_GUIDE.md

```
@@ -0,0 +1,671 @@
+   ```bash
+   git add .
+   git commit -m "Setup deployment configuration"
+   ```
+
+2. Push в main branch:
+   ```bash
+   git push origin main
+   ```
+
+3. Откройте GitHub Actions:
+   - https://github.com/Katsiarynakavaleuskaya/PulsePlate/actions
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)